<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommanderDataLog</class>
 <widget class="QWidget" name="CommanderDataLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommanderDataLog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Allgemein</string>
      </attribute>
      <widget class="QLCDNumber" name="lcdTime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdDate">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="numDigits">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Uhrzeit:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Datum:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;b&gt;Datensätze:&lt;/b&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>66</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;b&gt;Einheit:&lt;/b&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;b&gt;Abtastrate:&lt;/b&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblDataCount">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblDataRate">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblDataUnit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>290</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btInit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>181</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Datalogger Initialisieren</string>
       </property>
      </widget>
      <widget class="QComboBox" name="lbSerialportList">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>50</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SerialPort</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;b&gt;Gerätename:&lt;/b&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblDeviceName">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>130</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color : blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>130</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;b&gt;Gerätetyp:&lt;/b&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblDeviceType">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : blue;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btReadMem">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>250</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Speicherinfo</string>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>lcdTime</zorder>
      <zorder>lcdDate</zorder>
      <zorder>label_2</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>lblDataCount</zorder>
      <zorder>lblDataRate</zorder>
      <zorder>lblDataUnit</zorder>
      <zorder>btInit</zorder>
      <zorder>lbSerialportList</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>lblDeviceName</zorder>
      <zorder>label_9</zorder>
      <zorder>lblDeviceType</zorder>
      <zorder>btReadMem</zorder>
     </widget>
     <widget class="QWidget" name="tabDownload">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Download Data</string>
      </attribute>
      <widget class="QPushButton" name="btDataDownload">
       <property name="geometry">
        <rect>
         <x>497</x>
         <y>10</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>481</width>
         <height>301</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
      <widget class="QPushButton" name="btSave">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSetting">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 }</string>
      </property>
      <attribute name="title">
       <string>Logger Einstellung</string>
      </attribute>
      <widget class="QWidget" name="layoutWidgetA">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="gbDevice">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Gerät</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>551</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>LCD aktiv</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btDevieSettings">
              <property name="text">
               <string>Übernehmen</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbDateTime">
          <property name="title">
           <string>Datum / Uhrzeit</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbLogSetting">
          <property name="title">
           <string>Logverhalten</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>gbDateTime</zorder>
       <zorder>gbLogSetting</zorder>
       <zorder>gbDevice</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAlarm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 }</string>
      </property>
      <attribute name="title">
       <string>Alarm</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Temperatur</string>
         </property>
         <widget class="QCheckBox" name="cbTempObergrenze">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>30</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Obergrenze</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbTempUntergrenze">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>30</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Untergrenze</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbTempObergrenze">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbTempUntergrenze">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>60</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Luftfeuchtigkeit</string>
         </property>
         <widget class="QLineEdit" name="tbRelHObergrenze">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbRelHUntergrenze">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>60</y>
            <width>113</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbRelHObergrenze">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>30</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Obergrenze</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="cbRelHUntergrenze">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>30</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Untergrenze</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>208</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Übernehmen</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>188</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRealTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <attribute name="title">
       <string>Echtzeit</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hzBottom">
     <item>
      <spacer name="hzSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="btBeenden">
       <property name="text">
        <string>Beenden</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hzSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btBeenden</sender>
   <signal>clicked()</signal>
   <receiver>CommanderDataLog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
